src/integrations/settings-integration.php

```diff
@@ -24,7 +24,6 @@
 use Yoast\WP\SEO\Helpers\Taxonomy_Helper;
 use Yoast\WP\SEO\Helpers\User_Helper;
 use Yoast\WP\SEO\Helpers\Woocommerce_Helper;
-use Yoast\WP\SEO\Integrations\Admin\Social_Profiles_Helper;
```

Comment:
In this file on line 48 there is a reference to `wpseo_social` can this be removed now?

Reply:
No, that refers to our options framework' instance: WPSEO_Option_Social. In which we save the organization social profiles still, amongst other things.
We did not save the person social profiles to our options. We save(d) them in the user metadata.

Reply:
Cool :)